policyengine_taxsim/config/variable_mappings.yaml

```diff
@@ -465,6 +559,35 @@ policyengine_to_taxsim:
     group_order: 4
     full_text_group: "State Tax Calculation"
     group_column: 1
+    pre_simulation:
+      - ar:
+          implemented: true
+          pre_variable: ar_files_separately
+          variables:
+            - ar_income_tax_before_non_refundable_credits_indiv
+            - ar_income_tax_before_non_refundable_credits_joint
+      - mt:
+          implemented: true
+          pre_variable: mt_files_separately
+          variables:
+            - mt_income_tax_before_non_refundable_credits_indiv
+            - mt_income_tax_before_non_refundable_credits_joint
+      - ky:
+          implemented: true
+          pre_variable: ky_files_separately
+          variables:
+            - ky_income_tax_before_non_refundable_credits_indiv
+            - ky_income_tax_before_non_refundable_credits_joint
+    special_cases:
+      - ms:
+          implemented: false
+          variable: na_pe
+      - de:
```

Comment:
we need:
`de_income_tax_before_non_refundable_credits_indv`
`de_income_tax_before_non_refundable_credits_joint`

`ms_income_tax_before_credits_indiv`
`ms_income_tax_before_credits_joint`